fix: Special Characters "/" in Search Not Working Even When Added as … (#13188)
shopware/platform: changelog/_unreleased/2025-10-26-special-characters-in-search-not-working-even-when-added-as-preserved-characters-in-config-file.md

---

> title: Special Characters / in Search Not Working Even When Added as Preserved Characters in Config File

Reviewer: ```suggestion
title: Slash and backslash are now working correctly in search if configured as preserved characters
```

issue: 13179
---
# Core

> * Changed `tokenize` method in `Shopware\Core\Framework\DataAbstractionLayer\Search\Term\Tokenizer` to properly handle special characters such as `/`, `\`, etc., when they are specified as preserved characters in the configuration file.

Reviewer: ```suggestion
* Changed `tokenize` method in `Shopware\Core\Framework\DataAbstractionLayer\Search\Term\Tokenizer` to properly handle special characters `/` and `\`, when they are specified as preserved characters in the configuration file.
```

why "etc"? :sweat_smile: no other chars are allowed now, right?
